R/hypothesize.R

```diff
@@ -2,6 +2,8 @@
 #' @param x a data frame that can be coerced into a \code{\link[dplyr]{tbl_df}}
 #' @param null the null hypothesis. Options include "independence" and "point".
 #' @param ... arguments passed to downstream functions
+#' @return A tibble containing the response (and explanatory, if specified) variable data with
```

Comment:
We should have consistent formatting here - to period or not to period?

Reply:
Gonna remove the periods